python/tank/commands/setup_project.py

```diff
@@ -587,8 +587,14 @@ def _get_default_configuration_location(self, log, params):
                 
         location = {"darwin": None, "linux2": None, "win32": None}
         
-        # get the path to the primary storage  
-        primary_local_path = params.get_storage_path(constants.PRIMARY_STORAGE_NAME, sys.platform)        
+        # get the path to the primary storage
```

Comment:
i find this confusing. the comment is wrong because it is no longer the primary storage. How about something like 

```python
# get the path to the storage we want to use when calculating
# the default location for the installed config
if len(storage_names) == 1:
    # this config only has one storage so use that
    storage_name = storage_names[0]
else:
    # for multi root storages, you are required to have a primary storage
    # so we base the default config location off this
    storage_name = constants.PRIMARY_STORAGE_NAME

primary_local_path = params.get_storage_path(storage_name, sys.platform)
```

don't we need to update line 626 below too with similar logic?

Reply:
Updated the comment and the code around line 623 to use the chosen `storage_name`.